packages/migration/src/migrations/v0026_backgroundOverlayToOverlay.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function backgroundOverlayToOverlay({ pages, ...rest }) {
+  return {
+    pages: pages.map(reducePage),
+    ...rest,
+  };
+}
+
+function reducePage({ elements, ...rest }) {
+  return {
+    elements: elements.map(updateElement),
+    ...rest,
+  };
+}
+
+function updateElement(element) {
+  if (typeof element.backgroundOverlay !== 'undefined') {
```

Comment:
```suggestion
  if (element.backgroundOverlay !== 'undefined') {
```

Reply:
It's either `typeof element.backgroundOverlay !== 'undefined'` or `element.backgroundOverlay !== undefined` <-- without quotes. Both should work. One is checking the type, the other is checking the value directly. Since we're using `typeof` elsewhere as well in migrations, used that.